### ISSUE TITLE
GPGFinder won't find gpg if using GPGSuite (on macOS); panic: Couldn't find a suitable gpg executable

### ISSUE THREAD

**Issue by gabriel** [aptly-dev/aptly#887]
If using GPG Suite, which is common for macOS users, the gpg version lists:

```
gpg --version
gpg (GnuPG/MacGPG2) 2.2.17
libgcrypt 1.8.4
```

The GPGFinder is searching for:
```
expectedVersionSubstring: "(GnuPG) 2.",
```

so it fails to find it (since the substring is: `(GnuPG/MacGPG2) 2.`, which results in:
```
panic: Couldn't find a suitable gpg executable. Make sure gnupg2 is available as either gpg(v) or gpg(v)2 in $PATH
```

You can reproduce by installing GPGSuite using `brew cask install gpg-suite`.

Perhaps GPGFInder should allow the expected version substring to include MacGPG2?

**Comment by smira** [comment]
@gabriel don't have OS X to test with, but I can try adjusting the check to include that substring as well

**Comment by smira** [comment]
Fixed via #888

**Comment by smira** [comment]
`aptly_1.4.0+44+g24a0271` nightly release should be available now with the fix
